test/pipetool.uts

```diff
@@ -264,6 +264,8 @@ os.unlink("t2.pcap")
 = Test InjectSink and Inject3Sink
 ~ needs_root
 
+import mock
```

Comment:
Is this needed in this PR ?

Reply:
This is very small, A forgotten fix that provided from running the tests from the command line on windows. So small i thought i would just leave this in this PR.